pkg/occlient/occlient.go

```diff
@@ -1356,39 +1333,47 @@ func copyVolumesAndVolumeMounts(dc appsv1.DeploymentConfig, currentDC *appsv1.De
 
 // UpdateDCToGit replaces / updates the current DeplomentConfig with the appropriate
 // generated image from BuildConfig as well as the correct DeploymentConfig triggers for Git.
-func (c *Client) UpdateDCToGit(commonObjectMeta metav1.ObjectMeta, imageName string) error {
+func (c *Client) UpdateDCToGit(commonObjectMeta metav1.ObjectMeta, imageName string, ports []corev1.ContainerPort, componentSettings config.LocalConfigInfo, resourceLimits corev1.ResourceRequirements, envVars []corev1.EnvVar, isDeleteSupervisordVolumes bool, existingDC *appsv1.DeploymentConfig, existingCmpContainer corev1.Container, dcRollOutWaitCond dcRollOutWait) (err error) {
```

Comment:
any better way to fix the number of parameters in the function?

Reply:
agreed with @mik-dass this can't go in, we need a better way to do this

Reply:
Fixed now